apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/_components/EvaluationResults/EvaluationResultInfo/index.tsx

```diff
@@ -21,12 +22,15 @@ function useFetchDocumentLog({ documentLogId }: { documentLogId: number }) {
     data: documentLog,
     isLoading,
     error,
-  } = useSWR<DocumentLogWithMetadata>(
+  } = useSWR<DocumentLogWithMetadataAndError>(
```

Comment:
i find the naming is getting a bit crazy, could we keep it as it was and assume metadata includes error? 😅

Reply:
We have `DocumentLogsRepository` and `DocumentLogsWithErrorRepository`